clients/python-apiserver-client/README.md

```diff
@@ -0,0 +1,31 @@
+# Python client for KubeRay API server
+
+This Python client is currently only supporting Ray cluster mangement through usage of the `API server` Ray API. It implements all of the current functionality of the API server (and the operator) and provide pythonic APIs to the capabilities.
+
+The package supports well documented in the code Python objects that can be used to build and recieve payloads for creation, listing and delition of [template](https://ray-project.github.io/kuberay/components/apiserver/#compute-template) and Ray clusters.
+
+The main class of the package is [KubeRayAPIs](python_apiserver_client/kuberay_apis.py) that implements all of the functionality. It leverages [templates](python_apiserver_client/params/templates.py) and [cluster](python_apiserver_client/params/cluster.py) definitions, allowing to specify all required parameters as straight Python classes. Additional (intermediate) definitons are provided (see [environment variables](python_apiserver_client/params/environmentvariables.py), [volumes](python_apiserver_client/params/volumes.py), [head group](python_apiserver_client/params/headnode.py) and [worker group](python_apiserver_client/params/workernode.py))
+
+## Prerequisites
+
+It is also expected that the `kuberay operator` [Installation instructions are here.](https://github.com/ray-project/kuberay#quick-start) and `api server` [Installation instructions are here.](https://ray-project.github.io/kuberay/components/apiserver) are installed
+
+## installation
+
+Start from installing `setup-tools`
+
+```shell
+pip3 install -U setuptools
+```
+
+Now install our library locally. From the directory `path/to/kuberay/clients/python_apiserver_client` execute
+
+```shell
+pip3 install -e .
+```
+
+## Testing
+
+Test files [parameters_test](python_apiserver_client_test/api_params_test.py) and [api_test](python_apiserver_client_test/kuberay_api_test.py) exercise the package functionality and can also be used as a guide for API usage.
+
+Note that [api_test](python_apiserver_client_test/kuberay_api_test.py) requires installation of the package and creation of the additional [configmap](../../apiserver/test/job/code.yaml) in the default namespace
```

Comment:
Could you provide some instruction for running the test. I tested this way but I am not very certian.
```
# pip install pytest
# pip install python_apiserver_client-0.0.1-py3-none-any.whl
# test requires apiserver available at localhost:8888
# cd ../../apiserver && make run
pytest python_apiserver_client_test/kuberay_api_test.py
pytest python_apiserver_client_test/kuberay_api_test.py
```

Reply:
This is the correct approach. Although I was doing slightly differently.
instead of using pytest I was running tests directly from Intellij